federation/pkg/federatedtypes/crudtester/crudtester.go

```diff
@@ -29,6 +29,7 @@ import (
 
 const (
 	AnnotationTestFederationCRUDUpdate string = "federation.kubernetes.io/test-federation-crud-update"
+	ObjectNotExpectedWaitTime                 = 10 * time.Second
```

Comment:
Waiting too long is par for the course for negative tests, and waiting for too short an interval risks flakiness.  Will 10s be enough to ensure against running the tests in a resource-constrainted environment?